content/bigquery.textile

```diff
@@ -0,0 +1,109 @@
+---
+title: BigQuery rule
```

Comment:
I'm not sure what you're doing with the others, but tempted to say drop 'rule' from all of them. 

Also consider including GCP or Google like we do for AWS maybe?